h5py/_hl/files.py

```diff
@@ -396,6 +405,9 @@ def __init__(self, name, mode='r', driver=None,
             "aggregate"  Aggregators, VFD
             "none"       VFD
             If None use HDF5 defaults.
+        fs_page_size
+            File space page size in bytes. Only used when fs_strategy="page". If
+            None use HDF5 defaults.
```

Comment:
```suggestion
            None use HDF5 defaults (4096 bytes).
```

If you need to ensure `page_buf_size` is at least this big, what the default is is fairly important.

Reply:
We might also want to document that 512 is the minimum.

Reply:
Add the default value.